src/db/meta.rs

```diff
@@ -38,65 +40,106 @@ pub struct Meta {
 
     pub master_key_change_force: Option<isize>,
 
+    /// memory protection settings
     pub memory_protection: Option<MemoryProtection>,
 
+    /// custom icons
     pub custom_icons: CustomIcons,
 
+    /// whether the recycle bin is enabled
     pub recyclebin_enabled: Option<bool>,
 
     /// A UUID for the recycle bin group
     pub recyclebin_uuid: Option<String>,
 
+    /// last time the recycle bin was changed
     pub recyclebin_changed: Option<NaiveDateTime>,
 
+    /// UUID of the group containing entry templates
     pub entry_templates_group: Option<String>,
 
+    /// last time the group containing entry templates was changed
     pub entry_templates_group_changed: Option<NaiveDateTime>,
 
+    /// last selected group UUID
     pub last_selected_group: Option<String>,
 
     pub last_top_visible_group: Option<String>,
 
+    /// Maximum number of items of history to keep
     pub history_max_items: Option<usize>,
 
+    /// Maximum size of the history to keep
     pub history_max_size: Option<usize>,
 
+    /// Last time the settings were changed
     pub settings_changed: Option<NaiveDateTime>,
 
+    /// Binary attachments in the Metadata header
```

Comment:
@sseemayer do we have to mention that they are store in the Metadata header? That feels like an implementation detail. We could simply say `Binary attachments stored in the database`.

Reply:
I would like to keep it for now, since there are two different ways of specifying binary attachments in the file format, and users might be confused about that. The longer-term goal would be to consolidate into one attachment field.